### ISSUE TITLE
[FeatureHighlight][VoiceOver] Announce that a feature highlight has been presented

### ISSUE THREAD

**Issue by sigito** [material-components/material-components-ios#2677]
There is no context given to a user with VO turned on. If a user enters a screen with a feature highlight, there is no indication that a view is temporary and can be dismissed.`MDCFeatureHighlightViewController` should announce that there a feature highlighted.

**Comment by ianegordon** [comment]
Reminder that any text send to VO will need i18n support.

**Comment by romoore** [comment]
Closing as a duplicate of #4036 which has a bit more context.
